<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerProbeCalibrationModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerProbeCalibrationModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Display</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="Input">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string extracomment="Inputs for the module">Input</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="horizontalSpacing">
          <number>10</number>
         </property>
         <property name="verticalSpacing">
          <number>15</number>
         </property>
         <item row="7" column="0">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Selected Fiducials:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Rotations File:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Translation File:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="loadRotationsButton">
           <property name="text">
            <string>Load Rotations:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="loadTranslationsButton">
           <property name="text">
            <string>Load Translations</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Selected Images:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="qMRMLNodeComboBox" name="volumeMRMLNodeComboBox" native="true"/>
         </item>
         <item row="7" column="1">
          <widget class="qMRMLNodeComboBox" name="fiducialsMRMLNodeComboBox" native="true"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="rotationsLineEdit"/>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="translationsLineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="calibrateButton">
        <property name="text">
         <string>Calibrate US Probe</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="CalibrationBox">
        <property name="title">
         <string>Calibration Results</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>T1:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="t11Label">
           <property name="text">
            <string>t1.1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>T2:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QLabel" name="t21Label">
           <property name="text">
            <string>t2.1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="t12Label">
           <property name="text">
            <string>t1.2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="t13Label">
           <property name="text">
            <string>t1.3</string>
           </property>
          </widget>
         </item>
         <item row="0" column="8">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>W:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="9">
          <widget class="QLabel" name="w1Label">
           <property name="text">
            <string>w1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="7">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="6">
          <widget class="QLabel" name="t22Label">
           <property name="text">
            <string>t2.2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="6">
          <widget class="QLabel" name="t23Label">
           <property name="text">
            <string>t2.3</string>
           </property>
          </widget>
         </item>
         <item row="1" column="9">
          <widget class="QLabel" name="w2Label">
           <property name="text">
            <string>w2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="9">
          <widget class="QLabel" name="w3Label">
           <property name="text">
            <string>w3</string>
           </property>
          </widget>
         </item>
         <item row="0" column="10">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="11">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>S:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="12">
          <widget class="QLabel" name="sxLabel">
           <property name="text">
            <string>Sx</string>
           </property>
          </widget>
         </item>
         <item row="1" column="12">
          <widget class="QLabel" name="syLabel">
           <property name="text">
            <string>Sy</string>
           </property>
          </widget>
         </item>
         <item row="0" column="13">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <slots>
    <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
    <slot>setMRMLScene(vtkMRMLScene*)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
   <slots>
    <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerProbeCalibrationModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>fiducialsMRMLNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadTranslationsButton</sender>
   <signal>clicked()</signal>
   <receiver>qSlicerProbeCalibrationModuleWidget</receiver>
   <slot>loadTranslationsFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadRotationsButton</sender>
   <signal>clicked()</signal>
   <receiver>qSlicerProbeCalibrationModuleWidget</receiver>
   <slot>loadRotationsFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calibrateButton</sender>
   <signal>clicked()</signal>
   <receiver>qSlicerProbeCalibrationModuleWidget</receiver>
   <slot>calibrateUSProbe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerProbeCalibrationModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>volumeMRMLNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadTranslationsFile()</slot>
  <slot>loadRotationsFile()</slot>
  <slot>calibrateUSProbe()</slot>
 </slots>
</ui>
